extensions/blocks/donations/controls.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * External dependencies
```

Comment:
@automattic is considered external? :O

Reply:
:) Yes, as the source does not live in this repo.